home/App.tsx

```diff
@@ -9,16 +9,14 @@ import Store from './redux/Store';
 
 import './menu/DevMenuApp';
 
-export default (props: any) => {
```

Comment:
It looks like these "initial props" weren't used so I removed the propagation. Will this break anything @ide @brentvatne ?

Reply:
looking at the deleted file it seems like this is a safe change